Done project

volkova-lisa opened the pull request.

volkova-lisa committed:
Done project

ASKabanets commented on app/src/main/java/quizapp/volkova/openbuttons/MainActivity.kt:
```diff
@@ -1,11 +1,75 @@
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.app.ShareCompat
+
+
```
Remove empty line

ASKabanets commented on app/src/main/java/quizapp/volkova/openbuttons/MainActivity.kt:
```diff
@@ -1,11 +1,75 @@
+
+class MainActivity : AppCompatActivity(), View.OnClickListener {
+
+    lateinit var openLocationEditText: EditText
```
Add access modificators

ASKabanets commented on app/src/main/java/quizapp/volkova/openbuttons/MainActivity.kt:
```diff
@@ -1,11 +1,75 @@
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
+        openLocationEditText = findViewById<EditText>(R.id.location_edittext)
```
Try to use https://medium.com/androiddevelopers/use-view-binding-to-replace-findviewbyid-c83942471fc instead of `findViewById `

ASKabanets commented on app/src/main/java/quizapp/volkova/openbuttons/MainActivity.kt:
```diff
@@ -1,11 +1,75 @@
+        if (intent.resolveActivity(packageManager) != null) {
+            startActivity(intent)
+        } else {
+            Log.d("ImplicitIntents", "Can't handle this intent!")
```
Try to use https://github.com/JakeWharton/timber instead of simple Logs. Timber is kinda "industry standard"

ASKabanets commented on app/src/main/java/quizapp/volkova/openbuttons/MainActivity.kt:
```diff
@@ -1,11 +1,75 @@
+    }
+
+    fun openLocation(view: View?) {
+        val loc: String = openLocationEditText.getText().toString()
```
`loc ` -> `location` - unclear naming

ASKabanets commented on app/src/main/res/layout/activity_main.xml:
```diff
@@ -1,18 +1,51 @@
 
-    <TextView
+    <EditText
+        android:id="@+id/website_edittext"
```
`website_edittext ` -> `websiteEditText ` - Camel-case is preferable

ASKabanets reviewed (changes requested):
In general - ok. Minor comments. Also, `Done project ` unclear commit name

volkova-lisa committed:
changed findViewById to Binding,
importante!:
- buildFeatures bind true
- inflator inside onCreate
- bind.root after inflator

volkova-lisa committed:
changed findViewById to Binding,
importante!:
- buildFeatures bind true
- inflator inside onCreate
- bind.root after inflator

volkova-lisa committed:
fixed everything but camelcase in ids
